components/edge-functions/stripe-billing-portal.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Stripe Billing Portal
+description: Create secure billing portals for customers using Stripe's billing infrastructure, **powered by edge functions**.
+---
+
+<img src="/images/banners/stripe-billing-portal.png" className="rounded-lg" noZoom />
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Find it on GitHub: [<Icon icon="github" iconType="solid" /> /edgee-cloud/stripe-billing-portal-component](https://github.com/edgee-cloud/stripe-billing-portal-component)
+
+A WebAssembly component that integrates Stripe's Billing Portal directly at the edge. This component provides easy redirection to Stripe's Customer Portal, allowing customers to manage their subscriptions, payment methods, and billing information.
+
+## Features
+
+- **Stripe Billing Portal Integration**: Direct integration with Stripe's Customer Portal
+- **Simple Redirection**: Easy GET and POST endpoints for portal access
+- **Edge Performance**: Fast portal URL generation at the edge
+- **Flexible Configuration**: Support for custom return URLs
+- **WebAssembly Performance**: Secure, fast execution in a sandboxed environment
+- **Customer Management**: Handle customer portal sessions efficiently
+
+## Getting Started
+
+**To integrate Stripe Billing Portal with your project:**
+
+1. Set up a Stripe account and obtain your API key
+2. Open the Edgee console and navigate to your project's Components
+3. Select "Add a component" and choose `stripe-billing-portal` from the list of available edge functions
+4. Configure the following settings:
+   - **API Key**: Your Stripe API secret key
+   - **Return URL**: Default return URL after portal sessions (optional)
+   - **Path Configuration**: Set the billing portal endpoint path (e.g., `/stripe`)
+5. Click Save to activate the billing portal endpoint
+
+## Configuration Options
+
+### Required Settings
+- **API Key**: Your Stripe API secret key
+- **Endpoint Path**: The path for portal access (e.g., `/stripe`)
+
+### Optional Settings
+- **Return URL**: Default return URL after portal sessions
+
+### Setup Requirements
+- Active Stripe account
+- Stripe API secret key with billing permissions
+- Existing Stripe customers for portal access
+
+## Usage
```

Comment:
not a very useful section (and we don't really need to explain customer if/when they could use Stripe imho)